### ISSUE TITLE
Profile showcase issues

### ISSUE THREAD

**Issue by glitchminer** [status-im/status-desktop#12774]
# Bug Report

## **Communities**

**1. Ower of a community is displayed as "Admin" in list of communities**
![image](https://github.com/status-im/status-desktop/assets/50769329/5d279d4a-5b4a-42b2-8219-2eab45f2f459)

---
**2. Communities are only shown to members of that community or have previously visited/been invited**

---
**3. If user leaves a community and that community is shown it is not removed from the profile**

---
**4. Popup is not shown on first publish to showcase**
https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?type=design&node-id=14729-233295&mode=design&t=Puw7L99fopDCye5B-4
Requirement need clarified if it is to be shown once for each type (community/asset/etc) or only the very first time

---
**5. Name, image, description do not update on profile after update**

---
**6. Drag is being detected as a change before item is dropped**

https://github.com/status-im/status-desktop/assets/50769329/05de009c-54d6-41f6-8abb-cc39ace1b65a

---
**7. Communities in common are not shown in a separate section**

**8. Layout does not match Figma - icon is shown above name not on same line**

**9. Community permissions are not shown on tile**
![image](https://github.com/status-im/status-desktop/assets/50769329/559f343f-7c97-4ea0-90b1-7cd3ef205754)

---
10. When no communities are shared text should be "<Name> is keeping their community memberships private."

---
11. If the viewer of the community is not a member, clicking on the community does not navigate to it

---
12. Maximum number of communities is unspecified

---
13. Crown icon is shown depending on viewer's role, not the profile account's role 
(unclear if crown should be shown at all)

## **Accounts** 

14. Popup is not shown on first publish

---
15. Account name, color and emoji are not updated in profile on update

---
16. Text should read "<Name>'s accounts are private."  when no accounts are shared

---
17. Clicking on send opens the send dialog but it is not pre-populated with the address

---
18. Clicking on save does save the account but the names will not be significant after multiple "Status account" are added

---
19. Watch button has no tooltip

---
20. Clicking watch account button does nothing and is not shown in Figma

---
21. Removing an account from wallet that is shown in profile does not remove it from profile

---
22. Watch accounts are shown in list (requirement needs clarification on if this should be possible)

---
23. Max number of accounts supported is unknown?

## Restore / sync
24. Profile was not fetched when restoring using seed. Sync remains untested. 

## Collectibles and assets
Further dev work is planned, these were not tested.

### Additional Information

- Status desktop version: 0.15 RC2
- Operating System: Mac

**Comment by jrainville** [comment]
A couple of the issues are UI issues, so I assigned @caybro . Feel free to change the assignement @alexandraB99 if you think someone else should work on it.

I'll let @MishkaRogachev and whoever is assigned split the work depending on the UI vs messenger parts.

As for some of the issues found, @MishkaRogachev I'd tell you to ignore the ones about updating for now. Since it needs to send messages on waku, we need to think about how to make it work correctly.
So long story short, try to fix the ones that are simple fixes for now. We can track the rest in another ticket for later. We have more important things to fix than the Profile Showcase.

**Comment by MishkaRogachev** [comment]
@jrainville I'd rather wait for the updated requirements (end of next week I believe), there will be a number of changes that will solve some of the listed problems and even simplify some things

**Comment by caybro** [comment]
> A couple of the issues are UI issues, so I assigned @caybro . Feel free to change the assignement @alexandraB99 if you think someone else should work on it.

I'm afraid I won't have time to work on this in the near future (~1 month at least) due to the tasks in https://github.com/status-im/status-desktop/issues/12306

**Comment by MishkaRogachev** [comment]
@glitchminer i've opened a new epic for backend part: https://github.com/status-im/status-desktop/issues/13069
Because of new designs i think this issue can be closed

**Comment by iurimatias** [comment]
closing as designs have been updated
